### PR TITLE
Convert mouse clicks to driver calls

### PR DESCRIPTION
**Pull request by MatBebenek**
Clicking the mouse on the drawing panel results in drawing a line or moving the position of a device head.

### CODE REVIEW

**Comment by kowallus** [comment]
Is it finished? Update trello board please.

**Comment by MatBebenek** [comment]
Trello updated. Waiting for review

https://trello.com/c/voR76Nx9

File: src/main/java/edu/kis/powp/jobs2d/events/SelectClearPanelOptionListener.java [hunk 1]
```diff
@@ -2,12 +2,26 @@
...
 
+import javax.swing.*;
+
 public class SelectClearPanelOptionListener implements ActionListener {
-	@Override
-	public void actionPerformed(ActionEvent e) {
-		DrawerFeature.getDrawerController().clearPanel();
-	}
+
+    JPanel panel = DrawerFeature.getPanel();
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/events/SelectClearPanelOptionListener.java:13]: You have wrong code convention (change to tabs!)

File: src/main/java/edu/kis/powp/jobs2d/drivers/SelectMouseFigureOptionListener.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        int halfWindowWidth = drawPanel.getWidth() / 2;
+
+        this.drawPanel.addMouseListener(new MouseAdapter() {
+            @Override
+            public void mouseClicked(MouseEvent e) {
+                if (previousHeadXPosition == null || previousHeadYPosition == null) {
+                    updatePreviousMousePosition(e, halfWindowWidth, halfWindowHeight);
+                }
+
+                driverManager.getCurrentDriver().setPosition(previousHeadXPosition, previousHeadYPosition);
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/SelectMouseFigureOptionListener.java:41]: Please, make rightClick modify only position so only leftClick would actually draw.

File: src/main/java/edu/kis/powp/jobs2d/drivers/SelectMouseFigureOptionListener.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    public void actionPerformed(ActionEvent e) {
+        cleanHeadPosition();
+        int halfWindowHeight = drawPanel.getHeight() / 2;
+        int halfWindowWidth = drawPanel.getWidth() / 2;
+
+        this.drawPanel.addMouseListener(new MouseAdapter() {
+            @Override
+            public void mouseClicked(MouseEvent e) {
+                if (previousHeadXPosition == null || previousHeadYPosition == null) {
+                    updatePreviousMousePosition(e, halfWindowWidth, halfWindowHeight);
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/SelectMouseFigureOptionListener.java:38]: Maybe the first click should always only do SetPosition?

**Review by kowallus — CHANGES_REQUESTED** [review]
The code quality is ok. I have some minor suggestions and issues to fix.

**Comment by MatBebenek** [comment]
MR was updated as requested in comments.

**Comment by kowallus** [comment]
OK! You can claim a new task.

**Pull request merged**